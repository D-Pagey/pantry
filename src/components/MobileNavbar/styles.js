import styled from 'styled-components';

export const List = styled.ul`
    border: 1px solid red;
    display: flex;
    list-style: none;
    padding: 1rem 0;
    margin: 0;
    justify-content: space-evenly;
    max-width: 425px;
`;

export const Item = styled.li`
    color: blue;
`;
