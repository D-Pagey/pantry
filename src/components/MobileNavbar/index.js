import React from 'react';
import { NavLink } from 'react-router-dom';
import * as S from './styles';

const MobileNavbar = () => (
    <S.List>
        <S.Item>
            <NavLink to="/">Check Fridge</NavLink>
        </S.Item>
        <S.Item>
            <NavLink to="/add">Add Food</NavLink>
        </S.Item>
    </S.List>
);

export default MobileNavbar;
