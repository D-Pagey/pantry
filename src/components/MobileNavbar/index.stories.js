import React from 'react';
import { storiesOf } from '@storybook/react';
import MobileNavbar from '.';

const props = {};

storiesOf('MobileNavbar', module).add('default', () => <MobileNavbar {...props} />);
