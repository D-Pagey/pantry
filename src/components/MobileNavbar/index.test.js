import React from 'react';
import MobileNavbar from '.';

const props = {};

describe('MobileNavbar', () => {
    it('should render', () => {
        const { container } = render(<MobileNavbar {...props} />);
        expect(container.firstChild).toMatchSnapshot();
    });
});
