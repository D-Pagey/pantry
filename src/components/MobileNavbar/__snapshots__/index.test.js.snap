// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MobileNavbar should render 1`] = `
.c0 {
  border: 1px solid red;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  list-style: none;
  padding: 1rem 0;
  margin: 0;
  -webkit-box-pack: space-evenly;
  -webkit-justify-content: space-evenly;
  -ms-flex-pack: space-evenly;
  justify-content: space-evenly;
  max-width: 425px;
}

.c1 {
  color: blue;
}

<ul
  class="c0"
>
  <li
    class="c1"
  >
    <a
      aria-current="page"
      class="active"
      href="/"
    >
      Check Fridge
    </a>
  </li>
  <li
    class="c1"
  >
    <a
      href="/add"
    >
      Add Food
    </a>
  </li>
</ul>
`;
